<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>stu_1</class>
 <widget class="QDialog" name="stu_1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="back_label_1">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>1</y>
     <width>801</width>
     <height>601</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="back_pushbutton">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>470</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Add_Pushbutton">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>150</y>
     <width>139</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Add new Students</string>
   </property>
  </widget>
  <widget class="QPushButton" name="loadData">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>559</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Load data</string>
   </property>
  </widget>
  <widget class="QPushButton" name="stu">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>20</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="student_plus">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>20</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="home">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="course">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>20</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="course_plus">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>20</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>570</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>191</y>
     <width>351</width>
     <height>301</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
